spark-testing/spark_utils.py

```diff
@@ -175,6 +184,8 @@ def upload_mesos_test_jar():
 
 
 def dcos_test_jar_url():
+    _check_tests_assembly()
+
     if DCOS_SPARK_TEST_JAR_URL is None:
```

Comment:
I know this is out of scope for this PR, but this looks a bit strange. We now have a module that *fails* if `DCOS_SPARK_TEST_JAR_URL` is not truthy, and then explicitly check it here.

OK, no, I take it back. Our condition checks that at least one of `DCOS_SPARK_TEST_JAR_URL` or `DCOS_SPARK_TEST_JAR_PATH ` is valid.

Reply:
I was confused there for a minute too.